src/BloomExe/Book/UserPrefs.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Newtonsoft.Json;
+using System.IO;
+
+namespace Bloom.Book
+{
+	public class UserPrefs
+	{
+		private bool _loading = true;
+		private string _fileName;
+		private int _mostRecentPage;
+
+		public static UserPrefs Load(string fileName)
+		{
+			if (string.IsNullOrEmpty(fileName))
+				return null;
+
+			var userPrefs = File.Exists(fileName) ? JsonConvert.DeserializeObject<UserPrefs>(File.ReadAllText(fileName)) : new UserPrefs();
+			userPrefs._fileName = fileName;
+			userPrefs._loading = false;
```

Comment:
So how about removing that complication? (i.e., we don't need the _loading flag at all)